src/test/java/seedu/address/logic/commands/DeleteStudentFromTeamCommandTest.java

```diff
@@ -0,0 +1,217 @@
+        DeleteStudentFromTeamByEmailCommand deleteStudentFromTeamByEmailCommand =
+                new DeleteStudentFromTeamByEmailCommand(validPerson.getEmail(),
+                        newModule, tutorialClass, newTeam);
+
+        DeleteStudentFromTeamByIdCommand deleteStudentFromTeamByIdCommand =
+                new DeleteStudentFromTeamByIdCommand(validPerson.getStudentId(),
+                        newModule, tutorialClass, newTeam);
+
+
+        assertCommandFailure(deleteStudentFromTeamByIdCommand, model,
+                String.format(TeamMessages.MESSAGE_STUDENT_NOT_FOUND_IN_TEAM, Messages.format(validPerson),
+                        tutorialClass));
+
+        assertCommandFailure(deleteStudentFromTeamByEmailCommand, model,
+                String.format(TeamMessages.MESSAGE_STUDENT_NOT_FOUND_IN_TEAM, Messages.format(validPerson),
+                        tutorialClass));
+    }
+    @Test
+    public void invalidDeletionFromTeam_tutorialTeamNotExist_failure() {
+        TutorialTeam team = new TutorialTeam(VALID_TEAM_NAME_NEW);
+
+        DeleteStudentFromTeamByIdCommand deleteStudentFromTeamByIdCommand = new
+                DeleteStudentFromTeamByIdCommand(validPerson.getStudentId(), newModule, tutorialClass,
+                team);
+        DeleteStudentFromTeamByEmailCommand deleteStudentFromTeamByEmailCommand = new
+                DeleteStudentFromTeamByEmailCommand(validPerson.getEmail(), newModule, tutorialClass,
+                team);
+        assertCommandFailure(deleteStudentFromTeamByIdCommand, model,
+                String.format(TeamMessages.MESSAGE_TEAM_DOES_NOT_EXIST, team, tutorialClass));
+        assertCommandFailure(deleteStudentFromTeamByEmailCommand, model,
+                String.format(TeamMessages.MESSAGE_TEAM_DOES_NOT_EXIST, team, tutorialClass));
+    }
+
+    @Test
+    public void toString_test() {
+        tutorialClass.addStudent(validPerson);
+        DeleteStudentFromTeamByIdCommand deleteStudentFromTeamByIdCommand =
+                new DeleteStudentFromTeamByIdCommand(validPerson.getStudentId(), newModule, tutorialClass, newTeam);
+        DeleteStudentFromTeamByEmailCommand deleteOtherStudentFromTeamByEmailCommand =
+                new DeleteStudentFromTeamByEmailCommand(validOtherPerson.getEmail(),
+                        newModule, tutorialClass, newTeam);
+        DeleteStudentFromTeamByIndexCommand allocateStudentToTeamByIndexCommand =
+                new DeleteStudentFromTeamByIndexCommand(Index.fromZeroBased(0),
+                        newModule, tutorialClass, newTeam);
+        assertEquals(deleteOtherStudentFromTeamByEmailCommand.toString(),
+                deleteOtherStudentFromTeamByEmailCommand.toString());
+        assertEquals(deleteStudentFromTeamByIdCommand.toString(),
+                deleteStudentFromTeamByIdCommand.toString());
+        assertEquals(allocateStudentToTeamByIndexCommand.toString(),
+                allocateStudentToTeamByIndexCommand.toString());
```

Comment:
are these testcases meaningful? I think you are just comparing 2 same objects which should always be true anyways.

Reply:
I'm just following the test cases used in AB3. I also felt it wasnt necessary but I'm not sure if I should delete